DigitalOffice/Services/ProjectService/Repositories/ProjectRepository.cs

```diff
@@ -53,5 +54,32 @@ public Guid EditProjectById(DbProject dbProject)
 
             return dbProject.Id;
         }
+
+        public void DisableWorkersInProject(WorkersIdsInProjectRequest request)
+        {
+            DbProject dbProject = dbContext.Projects
+                .FirstOrDefault(p => p.Id == request.ProjectId);
+
+            if (dbProject == null)
+            {
+                throw new NullReferenceException("Project with this Id does not exist");
+            }
+
+            foreach (Guid workerId in request.WorkersIds)
+            {
+                DbProjectWorkerUser dbProjectWorker = dbProject.WorkersUsersIds?
+                    .FirstOrDefault(w => w.WorkerUserId == workerId);
+
+                if (dbProjectWorker == null)
+                {
+                    throw new NullReferenceException("Worker with this Id does not exist");
+                }
+
+                dbProjectWorker.IsActive = false;
+            }
+
+            dbContext.Projects.Update(dbProject);
+            dbContext.SaveChanges();
+        }
```

Comment:
Some might say that this is redundant. Since dbContext already tracks all the changes, "SaveChanges" call should be enough.
But! I think this is a good call to update the entity manually just to be sure.
Also I haven't thought of that algorithm yet: to find an entity by Id, change everything in it and pass the same model to Update() method. I like it.